PROJECT

```diff
@@ -11,19 +11,6 @@ plugins:
 projectName: integration-service
 repo: github.com/redhat-appstudio/integration-service
 resources:
-- api:
-    crdVersion: v1
-    namespaced: true
-  domain: redhat.com
-  group: appstudio
-  kind: IntegrationTestScenario
-  path: github.com/redhat-appstudio/integration-service/api/v1alpha1
-  version: v1alpha1
-- controller: true
-  domain: redhat.com
-  group: appstudio
-  kind: Integration
-  version: v1alpha1
```

Comment:
also conversion webhook from beta1 should be removed in this file

Reply:
Thanks, removed also.